docs/source/onboarding_resources/tutorial/index.ipynb

```diff
@@ -2,7 +2,17 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #76.                    "Wasting Between -3 SD and -2 SD (post-ensemble)",
</pre><p>Is this slow?  Have string comparisons in a call like this makes me nervous.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1606/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>

Reply:
<p>Hm, fair question. I could map the strings to integers on the comparator rather than the column side, which would be more performant.</p><p> </p><p>But I know from <a href="https://ihme.slack.com/archives/C014RDZA1GE/p1741388188575109" target="_blank">this profile of the simulation overall</a> that this is not a hotspot. So probably not going back and changing now.</p>